packages/training/book/src/development/debugging.md

```
@@ -0,0 +1,369 @@
+make train-12gb
+
+# Terminal 2: Monitor GPU
+watch -n 1 nvidia-smi
+
+# Look for:
+# - Training process: 40-60% memory
+# - vLLM process: 20-35% memory
+# - No OOM
+```
+
+### Check W&B Logs
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)